htdocs/api/index.php

```diff
@@ -434,4 +434,17 @@
 	echo $result;
 }
 
+// Now flush output buffers so that response data is sent to request client
+ob_end_flush();
+ob_flush();
```

Comment:
If we have a ob_end_flush before, this ob_flush should be useless, shouldn't it ?

Reply:
Agreed